client/src/components/Format/Format.tool.ts

```diff
@@ -0,0 +1,49 @@
+import { FormatValue } from '.'
+
+export type UnitOptions = {
+  value: number,
+  postfix?: string,
+  intlOptions?: Intl.NumberFormatOptions
+}[]
+
+export type UnitOption = {
```

Comment:
why `UnitOption` is different from `UnitOptions`? it should be like
`UnitOptions = UnitOption[]`